src/linux/link.rs

```diff
@@ -317,6 +317,7 @@ fn relink(elf_path: &Path, new_rpath: &[PathBuf]) -> Result<(), RelinkError> {
 }
 
 #[cfg(test)]
+#[cfg(target_os = "linux")]
```

Comment:
This isn't necessary for the `builtin` test. Part of the point is that it will make cross-compilation (a little) easier! :)